doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3330,6 +3330,43 @@ The fully expanded example above (without environment variables) looks like this
 
   curl "https://demo.dataverse.org/api/info/settings/incompleteMetadataViaApi"
 
+Get Zip File Download Limit
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Get the configured zip file download limit. The response contains the long value of the limit in bytes.
+
+This limit comes from the database setting :ref:`:ZipDownloadLimit` if set, or the default value if the database setting is not set, which is 104857600 (100MB).
+
+.. code-block:: bash
+
+  export SERVER_URL=https://demo.dataverse.org
+
+  curl "$SERVER_URL/api/info/zipDownloadLimit"
+
+The fully expanded example above (without environment variables) looks like this:
+
+.. code-block:: bash
+
+  curl "https://demo.dataverse.org/api/info/zipDownloadLimit"
+
+Show Support Of The Embargo Feature
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Know if the Dataverse instance has been configured to allow embargoes.
+
+The endpoint checks whether the database setting :ref:`:MaxEmbargoDurationInMonths`, which enables the embargo feature, has a value that enables the feature or not.
```

Comment:
In the future, will the SPA (or some other system) need to know the value of :MaxEmbargoDurationInMonths? If so, should we return it now, as part of the endpoint we're adding? Or would we add a second endpoint to get the number of months?

Reply:
I'm not sure, considering the embargo duration is something that is configured by the setting.

If necessary in the future, we can extend the endpoint to include the setting value or create a separate endpoint. 

For now I have preferred to keep it simple.

Reply:
I'm assuming the SPA will need to perform some validation when we get to the EDIT side of the dataset page. I can imagine an error like this:

"Please an embargo date before September 8, 2025" (for example)

Reply:
Yes, directly exposing the setting sounds good.